api/questions/views.py

```diff
@@ -102,31 +102,41 @@ def put(self, request, id, m_id):
                 error = {'error': 'The specified question does not exist'}
                 return Response(data=error, status=status.HTTP_404_NOT_FOUND)
             try:
-                request.data['created_by'] = request.user
-                is_authenticated = request.user
-                owner = Question.objects.filter(
-                    created_by_id=is_authenticated).first()
-                meetup_id = Question.objects.filter(meetup_id=m_id).first()
-
-                if owner:
+                qn = Question.objects.get(id=id)
```

Comment:
Hello @Philipotieno ,
A variable is a symbolic name for (or reference to) information. The variable's name represents what information the variable contains so it should be easy to notice it at a glance. 
You could consider updating the variable `qn` to an easy to notice variable name.
You can read [this](https://hackernoon.com/the-art-of-naming-variables-52f44de00aad) to learn more about such.

Reply:
Fixing that right now @Ogutu-Brian